### ISSUE TITLE
Add new type of format - Australian Highlander

### ISSUE THREAD

**Issue by spjspj** [magefree/mage#2458]
http://www.auseternal.com/7-point-highlander/

```
Australian highlander is a 60 card singleton format which uses the vintage ban list and a points system. The format has been played across Australia and New Zealand for 20+ years.

Only 7 points are allowed per deck. Points changes are made by the Points Committee, which announces any changes in line with Wizard’s ban/restriction announcements.
```

I've also heard of Canadian Highlander, which may be just a 100 card singleton format (not sure if pointed or not).

**Comment by fireshoes** [comment]
Canadian Highlander also uses points.
https://canadianhighlander.wordpress.com/rules-the-points-list-and-deck-construction/

**Comment by spjspj** [comment]
Possibly add an objective rating system based on examining the cards in say an EDH deck.
```
I was thinking more about the issue #2458 (basically in relation to edh which I've been playing quite a bit more of lately). You can build some totally unfair and ridiculous decks which take infinite turns etc (looking at narset etc). I built basically 20-free-turns Narset after getting trompled once too many times against my casual builds. Could we introduce a score to a given deck? Have an arbitrary point value - +4 points = any extra turn card, +3 points = Narset/Leovold/Meren/Azami, +5 points = any explicit land destruction, +1 point = any counters, any fog, +2 points = any buyback spells, +4 points= omniscience/enter the infinite, +3 points=splinter twin combos or similar. Being able to limit a table to objectively casual decks would lead to more fun games I think
```

**Comment by spjspj** [comment]
64ff81af753757ca24241d4051be265ac59b1e20 << hopefully done with this.  Rules that I could implement in the deck validator was - >= 100 cards, no sideboard, the current point list from fireshoe's url, 10 points per 100 cards (0 points for only a partial number of 100 cards) aka 100 cards = 10 points, 199 cards = 10 points, 200 cards = 20 points.

**Comment by spjspj** [comment]
Hopefully actually implemented with c98bf163be9a70b78bee7b658c2055a231a024b2 (#2838 & #2837).  Has the double mulligan rule on 6 (&5,4,3,2,1 cards).
